Build failure on arm{el,hf}: Incompatible pointer types in `ch_fuse.c`

Charliecloud 0.37 fails to build from source with the following error message on armel and armhf:
```
ch_fuse.c:68:19: error: initialization of ‘void (*)(struct fuse_req *, fuse_ino_t,  uint64_t)’ {aka ‘void (*)(struct fuse_req *, long long unsigned int,  long long unsigned int)’} from incompatible pointer type ‘void (*)(struct fuse_req *, fuse_ino_t,  long unsigned int)’ {aka ‘void (*)(struct fuse_req *, long long unsigned int,  long unsigned int)’} [-Werror=incompatible-pointer-types]
   68 |     .forget     = &sqfs_ll_op_forget,
      |                   ^
ch_fuse.c:68:19: note: (near initialization for ‘OPS.forget’)
```
Full build logs are available here:
* [armel](https://buildd.debian.org/status/fetch.php?pkg=charliecloud&arch=armel&ver=0.37-1&stamp=1709861674&file=log)
* [armhf](https://buildd.debian.org/status/fetch.php?pkg=charliecloud&arch=armhf&ver=0.37-1&stamp=1709893440&file=log)

The corresponding bug in the Debian bug tracker is [bug#1065793](https://bugs.debian.org/1065793).

--- a/bin/ch_fuse.c
+++ b/bin/ch_fuse.c
@@ -27,6 +27,11 @@
 // SquashFUSE redefines __le16 unless HAVE_LINUX_TYPES_LE16 is defined. We are
 // assuming it is defined in <linux/types.h> on your machine.
 #define HAVE_LINUX_TYPES_LE16
+// The forget operation in libfuse3 takes uint64_t as third parameter,
+// while SquashFUSE defaults to unsigned long as used in libfuse2.
+// This causes a mess on arches with different size of these types,
+// so explicitly switch to the libfuse3 variant.
+#define HAVE_FUSE_LL_FORGET_OP_64T
 // Now we can include ll.h.
 #include <squashfuse/ll.h>
 

